recipes/antlr4/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+cmake_minimum_required(VERSION 3.10)
+project(test_package CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(${PROJECT_NAME} main.cpp
```

Comment:
Would a smaller test_package make sense here?

Reply:
It could, if we could depend on executing Java and downloading https://www.antlr.org/download/antlr-4.8-complete.jar . This is the demo that is included with the C++ packages and all the files in the "generated" directories are actually generated.

Reply:
Alrighty that make sense thanks for confirming. It's a runtime so it's understandable